### PR TITLE
Update theme, gatsby, and yarn up

### PR DESCRIPTION
**Pull request by dshevtsov**
## Description

This pull request updates project dependencies: theme (4.7.11), gatsby (4.25.4), and other dependencies according to required restrictions (`yarn up`). Also, it updates the GH Pages workflow to print a correct link to the generated website from a fork.

## Motivation and Context

Use latest code before starting to work on Indexing automation.

## How Has This Been Tested?

Preview: https://magento-devdocs.github.io/commerce-php/

### CODE REVIEW

**Review by jeff-matthews — APPROVED** [review]
Looks good

Noticed that we didn't bump gatsby in the [commerce-webapi](https://github.com/AdobeDocs/commerce-webapi/blob/main/package.json#L11) repo when we updated the theme there. Not sure how much it matters, but it would be nice to use the same deps across all repos.

**Comment by dshevtsov** [comment]
> Noticed that we didn't bump gatsby in the [commerce-webapi](https://github.com/AdobeDocs/commerce-webapi/blob/main/package.json#L11) repo when we updated the theme there. Not sure how much it matters, but it would be nice to use the same deps across all repos.

At first, I wanted to update it there too, but since it was not necessary in scope of the issue I was working on, I decided to skip the update to avoid any unpredictable issues there.

In scope of the work with indexing, I want to experiment with the latest available features. When this one is finished, all other repos can be updated accordingly.

**Comment by dshevtsov** [comment]
Updated `CONTRIBUTING.md`. @jeff-matthews, please review.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -45,3 +45,18 @@ feel free to reach out to existing committers to have a conversation about that.
 ## Security Issues
 
 Security issues shouldn't be reported on this issue tracker. Instead, [file an issue to our security experts](https://helpx.adobe.com/security/alertus.html).
+
+## Site preview from a fork
+
+You can generate the website to view your changes using GitHub Pages.
+In your fork's Settings, make sure Actions are enabled and Pages are configured to deploy from the **root** folder of the **gh-pages** branch.
+
+1. Open the Actions tab
```
Comment (jeff-matthews) [inline .github/CONTRIBUTING.md:54]:
```suggestion
1. Click the **Actions** tab in your forked repo.
```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -45,3 +45,18 @@ feel free to reach out to existing committers to have a conversation about that.
...
+1. Choose GitHub Pages (side navigation menu on the left)
```
Comment (jeff-matthews) [inline .github/CONTRIBUTING.md:55]:
```suggestion
1. In the left navigation menu, click the **GitHub Pages** workflow.
```

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -45,3 +45,18 @@ feel free to reach out to existing committers to have a conversation about that.
...
+1. In the field `This workflow has a workflow_dispatch event trigger.`
```
Comment (jeff-matthews) [inline .github/CONTRIBUTING.md:56]:
```suggestion
1. In the upper-right of the page, click **Run workflow**.
```

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -45,3 +45,18 @@ feel free to reach out to existing committers to have a conversation about that.
...
+   - Open drop-down menu `Run workflow`
```
Comment (jeff-matthews) [inline .github/CONTRIBUTING.md:57]:
```suggestion
```

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -45,3 +45,18 @@ feel free to reach out to existing committers to have a conversation about that.
...
+   - Select the branch you want to preview as a website
```
Comment (jeff-matthews) [inline .github/CONTRIBUTING.md:58]:
```suggestion
   - Select the branch you want to preview as a website.
```

File: .github/CONTRIBUTING.md [hunk 6]
```diff
@@ -45,3 +45,18 @@ feel free to reach out to existing committers to have a conversation about that.
...
+   - **Run workflow**
```
Comment (jeff-matthews) [inline .github/CONTRIBUTING.md:59]:
```suggestion
   - Click **Run workflow**.
```

File: .github/CONTRIBUTING.md [hunk 7]
```diff
@@ -45,3 +45,18 @@ feel free to reach out to existing committers to have a conversation about that.
...
+   - A new run of **Github Pages** will appear in the table of the workflow runs
```
Comment (jeff-matthews) [inline .github/CONTRIBUTING.md:60]:
```suggestion
```

File: .github/CONTRIBUTING.md [hunk 8]
```diff
@@ -45,3 +45,18 @@ feel free to reach out to existing committers to have a conversation about that.
...
+1. When the workflow run is finished, **Github Pages** will be marked with a green check mark.
```
Comment (jeff-matthews) [inline .github/CONTRIBUTING.md:61]:
```suggestion
When the workflow run is finished, **Github Pages** will be marked with a green check mark. To identify the URL of the generated website, click **GitHub Pages** > **build-and-deploy** > **GH Pages URL**.
```

File: .github/CONTRIBUTING.md [hunk 9]
```diff
@@ -45,3 +45,18 @@ feel free to reach out to existing committers to have a conversation about that.
...
+1. For the URL of the generated website, navigate to the finished workflow's run **GitHub Pages** > **build-and-deploy** > **GH Pages URL**
```
Comment (jeff-matthews) [inline .github/CONTRIBUTING.md:62]:
```suggestion
```

File: .github/CONTRIBUTING.md [hunk 10]
```diff
@@ -45,3 +45,18 @@ feel free to reach out to existing committers to have a conversation about that.
 ## Security Issues
 
 Security issues shouldn't be reported on this issue tracker. Instead, [file an issue to our security experts](https://helpx.adobe.com/security/alertus.html).
+
+## Site preview from a fork
+
+You can generate the website to view your changes using GitHub Pages.
```
Comment (jeff-matthews) [inline .github/CONTRIBUTING.md:51]:
```suggestion
You can build and deploy changes from your branch to a test website using GitHub Pages.
```

File: .github/CONTRIBUTING.md [hunk 11]
```diff
@@ -45,3 +45,18 @@ feel free to reach out to existing committers to have a conversation about that.
...
+In your fork's Settings, make sure Actions are enabled and Pages are configured to deploy from the **root** folder of the **gh-pages** branch.
```
Comment (jeff-matthews) [inline .github/CONTRIBUTING.md:52]:
```suggestion
You must enable Actions in your forked repo settings and configure Pages to deploy from the **root** folder of the **gh-pages** branch.

To build and deploy your branch:
```

**Review by jeff-matthews — APPROVED** [review]
Suggestions are optional

**Pull request merged**